.vscode/settings.json

```diff
@@ -5,7 +5,7 @@
 	},
 	"[javascript]": {
 		"editor.codeActionsOnSave": {
-			"source.organizeImports": true
+			"source.organizeImports": "explicit"
```

Comment:
This snuck in here but explicit is the default and is technically the same as true so it's not likely to cause any upset: https://code.visualstudio.com/docs/typescript/typescript-refactoring#_code-actions-on-save